packages/roosterjs-content-model-dom/lib/domUtils/entityUtils.ts

```diff
@@ -107,12 +107,11 @@ function parseEntityClassName(
  * @param format The source entity format object
  * @returns A combined CSS class name string for entity wrapper
  */
-export function generateEntityClassNames(format: ContentModelEntityFormat): string {
+export function generateEntityClassList(format: ContentModelEntityFormat): string[] {
```

Comment:
Changing the function name may be a breaking change as it is already exported in the index file, suggest create a new function.